locales/nl.yml

```diff
@@ -370,7 +373,12 @@ nl:
               hebben we het *Enterprise* abonnement.
 
               Kijk anders even naar [het overzicht van alle abonnementen](/pricing.html).
-          # - q: What are the payment methods?
+
+          - q: Wat zijn de betaalmethoden?
+            a: |
+              We accepteren betalingen met de meeste credit cards (**Visa**, **MasterCard** and **American Express**).
+              We accepteren ook betalingen per factuur als het om grote teams gaat. Neem dan [contact](javascript:Intercom('show');) met ons op zodat we dit kunnen regelen.
```

Comment:
`Als het om grote teams gaat of er geen creditcard tot beschikking is, accepteren we ook betalingen per factuur.